frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -109,9 +109,11 @@ export const TopNavBar: React.FC<TopNavBarProps> = ({ className }) => {
                                 !user.isResearcher && (
                                 <NavbarStyledLink to="/studies">
                                     <Box
-                                        display='flex'
```

Comment:
You can use <Group> or <Stack> which have properties for these styles (so you don't have to define styles={{}})